sdk/src/settings/builder.rs

```diff
@@ -198,6 +201,7 @@ impl TryFrom<ClaimGeneratorInfoSettings> for ClaimGeneratorInfo {
 }
 
 /// Settings for an action template.
+#[cfg_attr(feature = "json_schema", derive(schemars::JsonSchema))]
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub(crate) struct ActionTemplateSettings {
```

Comment:
I'm tempted to leave actionTemplateSettings out or undocumented for now, since this is a lot to test and not in much use at the moment.